src/index.ts

```diff
@@ -419,6 +419,20 @@ class QueryPipeline extends EventEmitter {
         return obj;
       }, {});
 
+    let activePins = {} as Record<string, Set<string>>;
```

Comment:
A colon after the declaration is the typical way of declaring the type of a variable. The `as` syntax is used to coerce something of one type into another. Also, `const` over `let` since the variable is never being overwritten.
```
    const activePins: Record<string, Set<string>> = {};
```